README_GITFLEXIMOD.rst

```diff
@@ -54,17 +49,17 @@ when this is needed are:
 * After merging some other CTSM branch/tag into your currently
   checked-out branch
 
-**checkout_externals** must be run from the root of the source
+**./bin/git-fleximod** must be run from the root of the source
```

Comment:
This paragraph is not true and should be deleted, you can run git-fleximod from any subdirectory of the tree.